### DISCUSSION TITLE
Tried saving workspace... Workspace is now inaccessible?

### DISCUSSION THREAD

**Discussion by niamhkearney** [community/community#50451] [Codespaces]
### Select Topic Area

Bug

### Body

Hi guys I'm in a bit of a crisis. This is my first post here so apologies if I've made a mistake of some kind.

I've been using Codespaces to develop a daemon in c, and I figured I should try and save it just in case I get an error I can't figure out somewhere down the line or something and need a backup. I thought I'd try the File > Save Workspace As... option (I'm new to Codespaces so I have no idea if this was the proper way to go about this). 

The workspace it exported is 1KB big, and my codespace seems to have just... ceased to exist? 
![image](https://user-images.githubusercontent.com/71713529/226208359-d57640da-ff32-4372-83e4-a9fd27db7ffd.png)
![image](https://user-images.githubusercontent.com/71713529/226208388-0e9668ba-ae65-433d-a50d-b693c10215c1.png)

You can see at the top of the second image that my folders are right there, but when I click 'ok' it just writes underneath it "please select a file". 

Is there any hope of salvaging this? I do have a backup of it but I'd be losing a whole days work and I had _just_ finished the program. 

Any assistance would be greatly appreciated.

**Comment by niamhkearney** [comment]
Nevermind fellas - I went to File > Recent and fortunately it was there. I think it was trying to open the empty workspace I saved and just shat itself.

**Reply by osortega** [reply]
Hello @niamhkearney 👋 
Looks like you tried to save the file as a VS Code workspace file. [Workspace file documentation here], and then tried to open that workspace file.

To save a file you can do the same thing you do for VS Code Desktop which would be Cmd or Ctrl + s. 
If you want to go back to the folder you were working on before, you can go to `File > Open Folder... > /workspaces/<your-repo-name>/`.

Let me know if that helps 😃
